code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -18,51 +33,78 @@
 	mutanttongue = /obj/item/organ/tongue/vampire
 	limbs_id = "human"
 	skinned_type = /obj/item/stack/sheet/animalhide/human
+	///some starter text sent to the vampire initially, because vampires have shit to do to stay alive
 	var/info_text = "You are a <span class='danger'>Vampire</span>. You will slowly but constantly lose blood if outside of a coffin. If inside a coffin, you will slowly heal. You may gain more blood by grabbing a live victim and using your drain ability."
-	var/obj/effect/proc_holder/spell/targeted/shapeshift/bat/batform //attached to the datum itself to avoid cloning memes, and other duplicates
+	///static list shared among all vampire species datums that give a house name for each department
+	var/static/list/vampire_houses = list(
```

Comment:
I don't understand. Why does this start out filled at all? It looks like it's a mixed list, which is a bad sign.

Reply:
good point, it could be empty